gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
+          compilation_level: 'ADVANCED',
+          warning_level: 'VERBOSE',
+          language_in: 'ECMASCRIPT6_STRICT',
+          language_out: 'ECMASCRIPT5_STRICT',
+          output_wrapper: '(function(){\n%output%\n}).call(this)',
+          externs: 'externs/html5.js',
+          js_output_file: 'CustomElementsV1.min.js'
+        }))
+      .pipe(gulp.dest('./dist'));
+});
+
+gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements',
```

Comment:
Build integration with the rest of the polyfills, TBD (see the above thread with @addyosmani).

I would actually like to see the polyfills separated. To me it's clearer, and given a mix of environments with different needs, I think it's easier to know what to include. To use Polymer on Safari, for instance, we'll only need Custom Elements and HTML Imports. On older versions of Chrome, just Custom Elements.